docs/src/validator/best-practices/monitoring.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Validator Monitoring Best Practices
+sidebar_label: Monitoring
+---
+
+It is essential that you have monitoring in place on your validator. In the event that your validator is delinquent (behind the rest of the network) you want to respond immediately to fix the issue. One very useful tool to monitor your validator is `solana-watchtower`.
+
+## Solana Watchtower
+
+Solana Watchtower is an extremely useful monitoring tool that will regularly monitor the health of your validator. It can monitor your validator for delinquency then notify you on your application of choice: Slack, Discord, Telegram or Twilio. Additionally, `solana-watchtower` has the ability to monitor the health of the entire cluster so that you can be aware of any cluster wide problems.
+
+### Getting Started
+
+To get started with Solana Watchtower, run `solana-watchtower --help`. From the help menu, you can see the optional flags and an explanation of the command.
+
+Here is a sample command that will monitor a validator node with an identity public key of `2uTk98rqqwENevkPH2AHHzGHXgeGc1h6ku8hQUqWeXZp`:
+
+```
+solana-watchtower --monitor-active-stake --validator-identity \
+  2uTk98rqqwENevkPH2AHHzGHXgeGc1h6ku8hQUqWeXZp
+```
+
+The command will monitor your validator, but you will not get notifications unless you added the environment variables mentioned in `solana-watchtower --help`. Since getting each of these services setup for notifications is not straight forward, the next section will walk through setting up watchtower notifications on Telegram.
+
+### Best Practices
+
+It is a best practice to run the `solana-watchtower` command on a separate server from your validator.
+
+In the case that you run `solana-watchtower` on the same computer as your `solana-validator` process, then during catastrophic events like a power outage, you will not be aware of the issue, because your `solana-watchtower` process will stop at the same time as your `solana-validator` process.
+
+Additionally, while running the `solana-watchtower` process manually with environment variables set in the terminal is a good way to test out the command, it is not operationally sound because the process will not be restarted when the terminal closes or during a system restart.
+
+Instead, you could run your `solana-watchtower` command as a system process similar to `solana-validator`. In the system process file, you can specify the environment variables for your bot.
+
+### Setup Telegram Notifications
```

Comment:
Recommend adding section on PagerDuty integration as well, maybe before Telegram, as this is a higher fidelity and more "built for purpose" devops tool. PagerDuty integration is available in 1.16.0 and master, however as watchtower runs on a separate machine using 1.16.0 shouldn't be an issue

Reply:
Will add in a separate PR. Adding a task.